client/hotrod-client-new/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java

```diff
@@ -9,19 +9,29 @@
 import java.util.Random;
 import java.util.concurrent.CompletableFuture;
 
+import org.infinispan.client.hotrod.AuthenticationTest;
 import org.infinispan.client.hotrod.RemoteCache;
 import org.infinispan.client.hotrod.test.SingleHotRodServerTest;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.protostream.SerializationContextInitializer;
 import org.infinispan.query.model.Sale;
+import org.infinispan.server.core.test.ServerTestingUtil;
+import org.infinispan.server.hotrod.HotRodServer;
+import org.infinispan.server.hotrod.configuration.HotRodServerConfigurationBuilder;
+import org.infinispan.server.hotrod.test.HotRodTestingUtil;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "org.infinispan.client.hotrod.query.pressure.LargePutAllPressureTest")
 public class LargePutAllPressureTest extends SingleHotRodServerTest {
 
-   private final static int SIZE = 15_000;
+   private static final Log log = LogFactory.getLog(AuthenticationTest.class);
+
+   private static final int SIZE = 15_000;
+   private static final int TIMEOUT = 1_000 * 60; // 1 minute
```

Comment:
I would recommend doing TimeUnit.MINUTES.toMillis(1) in the future. But I won't have that stop merging.